res/controllers/Denon-MC7000-scripts.js

```diff
@@ -1188,15 +1157,7 @@ MC7000.StarsUp = function(channel, control, value, status, group) {
     } else {
         if (MC7000.PADModePitch[deckNumber]) {
             for (var i = 1; i <= 8; i++) {
-                if (deckNumber === 1) {
-                    MC7000.halftoneToPadMap1[i-1] = MC7000.halftoneToPadMap1[i-1] + 8; // pitch up
-                } else if (deckNumber === 2) {
-                    MC7000.halftoneToPadMap2[i-1] = MC7000.halftoneToPadMap2[i-1] + 8; // pitch up
-                } else if (deckNumber === 3) {
-                    MC7000.halftoneToPadMap3[i-1] = MC7000.halftoneToPadMap3[i-1] + 8; // pitch up
-                } else if (deckNumber === 4) {
-                    MC7000.halftoneToPadMap4[i-1] = MC7000.halftoneToPadMap4[i-1] + 8; // pitch up
-                }
+                MC7000.halftoneToPadMap[deckNumber-1][i-1] = MC7000.halftoneToPadMap[deckNumber-1][i-1] + 8; // pitch up
```

Comment:
instead of starting at 1 and subtracting 1 everwhere, just start at 0 and count until 8:
```js
for (var i = 0; i < 8; i++) {
    MC7000.halftoneToPadMap[deckNumber-1][i] = MC7000.halftoneToPadMap[deckNumber-1][i] + 8; // pitch up
```

Reply:
done